apps/core/src/dashboards/dashboards.controller.ts

```diff
@@ -112,10 +114,41 @@ export class DashboardsController {
       throw either.err;
     }
 
-    if (!either.ok) {
+    if (isNothing(either.ok)) {
       throw new NotFoundException();
     }
 
     return either.ok;
   }
+
+  @HttpCode(200)
+  @Delete('bulk')
+  public async bulkDelete(
+    @Body() bulkDeleteDashboardsDto: BulkDeleteDashboardDto,
+  ): Promise<{ deletedIds: Dashboard['id'][] }> {
+    // cache is invalid after deleting a dashboard
+    await this.cacheManager.reset();
+
+    const eithers = await this.dashboardsService.bulkDelete(
```

Comment:
What's the reasoning behind calling this `eithers`? It is kinda hard to read for me, I would expect it to be something like `response`.

Reply:
Yeah, this is a mistake. It should be `result`. Would that be a better name?

Reply:
Yes, thanks.